pkg/integration/tests/branch/create_tag.go

```diff
@@ -0,0 +1,47 @@
+package branch
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var CreateTag = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Create a new tag on branch",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.
+			CreateNCommits(10).
+			NewBranch("new-branch").
+			EmptyCommit("new commit")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		t.Views().Branches().
+			Focus().
+			Lines(
+				MatchesRegexp(`\*\s*new-branch`).IsSelected(),
+				MatchesRegexp(`master`),
+			).
+			SelectNextItem().
+			Press(keys.Branches.CreateTag)
+
+		t.ExpectPopup().Menu().
+			Title(Equals("Create tag")).
+			Confirm()
+
+		t.ExpectPopup().Prompt().
+			Title(Equals("Tag name:")).
+			Type("new-tag").
+			Confirm()
+
+		t.Views().Tags().
```

Comment:
```suggestion
		t.Views().Tags().Focus().
```
Adding the focus to mimic user interaction, and also cos in the future we might not render to hidden views